apps/indexer/src/eventHandlers/voting.ts

```diff
@@ -64,8 +65,10 @@ export const voteCast = async (
 export const proposalCreated = async (
   context: Context,
   daoId: string,
+  blockTime: number,
   args: {
     proposalId: string;
+    txHash: Hex;
     proposer: Address;
```

Comment:
Potential division by zero error. If blockDelta is 0 (start and end block are the same), the endTimestamp calculation will be timestamp + 0, which might not be the intended behavior and could cause issues with proposal timing logic.